solr/core/src/java/org/apache/solr/cloud/api/collections/OverseerCollectionMessageHandler.java

```diff
@@ -255,7 +255,7 @@ public OverseerSolrResponse processMessage(ZkNodeProps message, String operation
     MDCLoggingContext.setCollection(message.getStr(COLLECTION));
     MDCLoggingContext.setShard(message.getStr(SHARD_ID_PROP));
     MDCLoggingContext.setReplica(message.getStr(REPLICA_PROP));
-    log.debug("OverseerCollectionMessageHandler.processMessage : {} , {}", operation, message);
```

Comment:
this can make it too verbose

Reply:
@noblepaul I share similar concern as you! 

At the end I still made such a change as this is around the same level as processing message from `overseer/queue`, which we have already been printing very similar message https://github.com/fullstorydev/lucene-solr/blob/75e89929eb360b513ee864aeb23a80c049747246/solr/core/src/java/org/apache/solr/cloud/Overseer.java#L258